mathCalc/Friedman.cpp

```diff
@@ -0,0 +1,116 @@
+#include "Headers.h"
+
+Friedman::Friedman(std::vector<std::vector<double>> &vec)
+{
+	for (size_t i = 0; i < vec.size(); i++)
+	{
+		dataVector.push_back(vec[i]);
+	}
+	m_dF = dataVector.size() - 1;
+}
+
+
+bool Friedman::signRanks()
+{
+	if (m_dF > 50)
```

Comment:
to powinno byc w .h static const size_t max... a nie z dupy nagle 50 w kodzie zaszyte